.github/workflows/linters.yml

```diff
@@ -0,0 +1,44 @@
+name: Linters
```

Comment:
- [ ] Please At the top of the page the user should see the total amount for the category (the sum of all of the amounts of the transactions in that category)
![image](https://user-images.githubusercontent.com/74744787/219891846-4c536228-a80f-4df3-9ae4-6984150ca34a.png)